src/refiners/fluxion/layers/module.py

```diff
@@ -164,6 +164,10 @@ def device(self) -> Device:
     def dtype(self) -> DType:
         return self.weight.dtype
 
+    def __repr__(self) -> str:
+        str = super().__repr__()
+        return f"{str} [device={self.device}, dtype={self.dtype}]"
+
```

Comment:
For it to show up in the error tree, you need to modify `__str__` 
```suggestion
    def __str__(self) -> str:
        return f"{super().__str__().removesuffix(')')}, device={self.device}, dtype={str(self.dtype).removeprefix('torch.')})"

```